recommendations/data-summary.md

```diff
@@ -0,0 +1,3 @@
+- Something went wrong with the data analysis
+- Try running the benchmark for a longer time
+– If the issue persists, open an issue at https://github.com/nearform/node-clinic-doctor
```

Comment:
this dash is a different character, causing it to collapse into the previous line:
![image](https://user-images.githubusercontent.com/1006268/62531822-7f535c80-b843-11e9-9c1a-9dcc531358d4.png)
```suggestion
- If the issue persists, open an issue at https://github.com/nearform/node-clinic-doctor
```